plaso/storage/file_interface.py

```diff
@@ -308,6 +308,25 @@ def GetEventDataByIdentifier(self, identifier):
     """
     return self._storage_file.GetEventDataByIdentifier(identifier)
 
+  def GetEventDataStream(self):
+    """Retrieves the event data stream.
```

Comment:
Change to "Retrieves the event data streams". Do this for all other implementations of this method as well.

Reply:
done